corehq/apps/oauth_integrations/migrations/0004_auto_20220304_1139.py

```diff
@@ -0,0 +1,24 @@
+# Generated by Django 2.2.27 on 2022-03-04 11:39
```

Comment:
I feel like in the future we should ensure we introduce the minimum amount of migrations necessary. Could this migration have been merged with a previous one? something to keep in mind in the future :)

cc @AmitPhulera

Reply:
Ohh! I did not notice that during the review. I will be mindful of it in the future. Thanks for looking :)